HHS/Head-Start-TTADP
# [TTAHUB-3102] Temporarily disable resource scrape job

## Description of change

Temporarily disable the resource scrape job.

**NOTE:** This isn't something we are moving forward with ATM just want to have it reviewed in case we decide to move forward.

## How to test

- Make sure all tests pass and you can add resources to objectives without issue.

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-3102


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
